packages/gnushogi/build.sh

```diff
@@ -2,22 +2,39 @@ TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/gnushogi/
 TERMUX_PKG_DESCRIPTION="Program that plays the game of Shogi, also known as Japanese Chess"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.4.2
-TERMUX_PKG_REVISION=4
-TERMUX_PKG_SRCURL=https://mirrors.kernel.org/gnu/gnushogi/gnushogi-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=1ecc48a866303c63652552b325d685e7ef5e9893244080291a61d96505d52b29
+_COMMIT=5bb0b5b2f6953b3250e965c7ecaf108215751a74
+TERMUX_PKG_VERSION=2014.11.19
```

Comment:
I'd prefer `1.4.2-p20141119`.

Reply:
Sure, I have changed it to that

Reply:
It appears that if I use the version format you requested, I cannot also use the same implementation of `termux_step_post_get_source()` that packages xtkoba edited have, so I had to modify that. Maybe that part doesn't matter very much.

Reply:
@licy183 is the way it is now close to what you meant by changing the version format?